code/modules/assembly/igniter.dm

```diff
@@ -22,7 +22,7 @@
 			if (istype(src.loc.loc, /obj/item/weapon/reagent_containers/glass/beaker/))
 				var/obj/item/weapon/reagent_containers/glass/beaker/beakerbomb = src.loc.loc
 				if(beakerbomb)
-					beakerbomb.ignite()
+					beakerbomb.fire_act()

```

Comment:
this is supposed to be ignite--as it's a special proc only for beakers--if you need to rename it, that's fine, but it's definitely not supposed to be fire_act